include/xgboost/tree_model.h

```diff
@@ -143,7 +160,7 @@ class TreeModel {
      * \param split_cond  split condition
      * \param default_left the default direction when feature is unknown
      */
-    inline void SetSplit(unsigned split_index, TSplitCond split_cond,
+    inline void SetSplit(unsigned split_index, SplitCondT split_cond,
```

Comment:
The `inline` is not useful here.  Any methods defined inside class is automatically inlined.